mwdb/core/karton.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import shutil
+import tempfile
+
+from flask import g
+from karton.core import Config as KartonConfig
+from karton.core import Producer, Resource, Task
+from karton.core.backend import KartonBackend
+from karton.core.inspect import KartonState
+from karton.core.task import TaskPriority
+
+from .config import app_config
+
+logger = logging.getLogger("mwdb.karton")
+
+
+def get_karton_producer() -> Producer:
+    return Producer(
+        identity="karton.mwdb", config=KartonConfig(app_config.karton.config_path)
+    )
+
+
+def send_file_to_karton(file):
+    from mwdb.model.file import File
+
+    try:
+        path = file.get_path()
+        tmpfile = None
+    except Exception:
```

Comment:
`Exception` seems kinda broad

Reply:
Changed and improved exception handling a bit (to be sure that `timpfile` is closed on unhandled exception)

As in comment, that part is planned to be simplified further with file-like object support in `karton.Resource`.